app/app.py

```diff
@@ -1,15 +1,12 @@
-import datetime
 import logging
-from typing import Any
 
-import pandas as pd
-import plotly.express as px
-import psycopg2
 from dash import Dash, dcc, html
 from flask import Flask
 
 from app.config.app_config import app_config
 from app.config.logging_config import configure_logging
+from app.services.dashboard_service import FigureService
```

Comment:
isn't `dashboard_service.py` now renamed to `figure_service.py` ?

Reply:
Good spot! I'll fix it on the next PR 🦅 👁️ 😁